core/templates/components/certificate-assessment-offering-helper/certificate-offering-details.component.html

```
@@ -1,12 +1,152 @@
+      </div>
+
+      <button class="oppia-certificate-outcome-add" type="button" (click)="addOutcome()">
+        <span aria-hidden="true">+</span> Add another outcome
+      </button>
+    </div>
+  </div>
 
   <div class="oppia-certificate-offering-details-actions">
-    <button (click)="onCancelClicked()">Cancel</button>
-    <button (click)="onNextClicked()">Next</button>
+    <button class="e2e-test-certificate-offering-details-next-button oppia-certificate-btn-next"
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)